src/inmanta/module.py

```diff
@@ -2279,6 +2282,15 @@ def requires(self) -> "List[InmantaModuleRequirement]":
             reqs.append(reqe)
         return reqs
 
+    def log_install_information(self, project: "Project", module_name: str) -> None:
+        """
+        Display information about this module's installation
+        :param project: The project associated with the module.
+        :param module_name: The module's name.
+        """
+        source = f" from {project.downloadpath}" if self.GENERATION == ModuleGeneration.V1 else ""
+        LOGGER.debug(f"Installing module {module_name} (v{self.GENERATION.value}) version {self.version}{source}.")
```

Comment:
Perhaps we could include the constraints as additional debugging information? e.g. "Installing module mymod (v2) version 1.0.0 (from constraints mymod~=1.0,mymod>=1.0.0)."

Reply:
The `module_spec` argument of `ModuleSource.install` is a list of `Requirement` instances on that module. You should be able to easily extract the set of constraints from that, e.g. in `log_pre_install_information`.